internal/host/validator.go

```diff
@@ -395,15 +395,24 @@ func (v *validator) isValidPlatform(c *validationContext) ValidationStatus {
 	if c.inventory.SystemVendor == nil {
 		return ValidationError
 	}
-	return boolValue(!funk.ContainsString(invalidPlatforms, c.inventory.SystemVendor.ProductName))
+	if funk.ContainsString(invalidPlatforms, c.inventory.SystemVendor.ProductName) {
+		// In case there is no cluster validation is pending
+		if c.cluster == nil {
+			return ValidationPending
+		} else {
+			//In case userManagedNetworking is true, we don't care about the platform
+			return boolValue(swag.BoolValue(c.cluster.UserManagedNetworking))
+		}
+	}
+	return boolValue(true)
 }
 
 func (v *validator) printValidPlatform(c *validationContext, status ValidationStatus) string {
 	switch status {
 	case ValidationSuccess:
 		return fmt.Sprintf("Platform %s is allowed", c.inventory.SystemVendor.ProductName)
 	case ValidationFailure:
-		return fmt.Sprintf("Platform %s is forbidden", c.inventory.SystemVendor.ProductName)
+		return fmt.Sprintf("Platform %s is forbidden, the platform is supported only for user managed networking", c.inventory.SystemVendor.ProductName)
```

Comment:
Simplify sentence, and also don't assume that users know the relationship between SNO and user-managed networking.

```suggestion
		return fmt.Sprintf("Platform %s is only supported for Single Node OpenShift or user-managed networking", c.inventory.SystemVendor.ProductName)
```